src/main.cpp

```diff
@@ -1,10 +1,14 @@
+#include "IO_Interface.hpp"
```

Comment:
Please write the filenames in the naming format specified on the [wiki](https://github.com/R2D2-2018/R2D2-2018/wiki/Code-standard).